src/main/java/org/apache/commons/imaging/formats/tiff/TiffImageParser.java

```diff
@@ -898,7 +921,7 @@ TiffRasterData getFloatingPointRasterData(
 
         final ImageDataReader dataReader = imageData.getDataReader(directory,
             photometricInterpreter, bitsPerPixel, bitsPerSample, predictor,
-            samplesPerPixel, width, height, compression, byteOrder);
+          samplesPerPixel, width, height, compression, 1, byteOrder);
```

Comment:
Is `1` the equivalent of `TiffTagConstants.PLANAR_CONFIGURATION_VALUE_CHUNKY`? If so, I think we should use the constant?

Reply:
Agreed. Good catch.